src/main/resources/bootstrap.yaml

```diff
@@ -6,6 +6,6 @@ spring:
       paths: /mnt/secrets/reform-scan
       aliases:
         app-insights-instrumentation-key: azure.application-insights.instrumentation-key
-        storage-account-name: STORAGE_ACCOUNT_NAME
-        storage-account-primary-key: STORAGE_ACCOUNT_KEY
-        storage-account-secondary-key: STORAGE_ACCOUNT_SECONDARY_KEY
+        storage-account-name: storage.account_name
+        storage-account-primary-key:  storage.account_key
+        storage-account-secondary-key:  storage.account_secondary_key
```

Comment:
Hmm.. not sure this change is needed 🤔

Reply:
Existing one didn't work so assume this would work. We will know now as the defaults have been removed